src/test/scala/medeia/generic/GenericDecoderSpec.scala

```diff
@@ -19,17 +19,20 @@ class GenericDecoderSpec extends MedeiaSpec {
   it should "decode empty values to None" in {
     case class Simple(int: Option[Int])
     val doc = BsonDocument()
-    doc.fromBson[Simple].right.value should ===(Simple(None))
+    val decoder = semiauto.deriveBsonDecoder[Simple]
+    decoder.decode(doc).right.value should ===(Simple(None))
   }
 
   it should "fail gracefully on nested decoders" in {
+    import medeia.generic.auto._
```

Comment:
I really hate this but: Currently when importig medeia.generic.auto it will use shapeless to derive encoders for List and Option and such. I tried to fiddle something that gives the DefaultInstandes precedence over the Shapeless stuff but I failed...
@markus1189 ideas?